ntoskrnl/se/srm.c

```diff
@@ -376,17 +381,100 @@ SepRmCreateLogonSession(
     return Status;
 }
 
+/**
+ * @brief
+ * Deletes a logon session from the logon sessions database.
+ * 
+ * @param[in] LogonLuid
+ * A logon ID represented as a LUID. This LUID is used to point
+ * the exact logon session saved within the database.
+ * 
+ * @return
+ * STATUS_SUCCESS is returned if the logon session has been deleted successfully.
+ * STATUS_NO_SUCH_LOGON_SESSION is returned if the logon session with the submitted
+ * LUID doesn't exist. STATUS_BAD_LOGON_SESSION_STATE is returned if the logon session
+ * is still in use and we're not allowed to delete it. Otherwise a NTSTATUS failure code
+ * is returned.
+ */
 static
 NTSTATUS
 SepRmDeleteLogonSession(
-    PLUID LogonLuid)
+    _In_ PLUID LogonLuid)
```

Comment:
As far as I'm concerned, yes. Windows doesn't check if the logon ID provided matches with one of these two internal logon sessions, presumably because they'd never pass such LUIDs to that function. Although we can do way better than that, I could add a check myself for it just in case but I'd like to hear you an opinion from this.

Reply:
> As far as I'm concerned, yes. Windows doesn't check if the logon ID provided matches with one of these two internal logon sessions

And how do you know that? :trollface: Have you written a test that validates this?